### PR TITLE
MEG-BIDS examples: update ds000117 from Rik

### PR DESCRIPTION
**Pull request by guiomar**

### CODE REVIEW

**Comment by guiomar** [comment]
@jasmainak I've realized the validator doen't check predefined fields. For example, Rik has in the coordinate system in one field "MEGCoordinateSystem": "Neuromag/Elekta" and in other: "AnatomicalLandmarkCoordinateSystem": "Elekta/Neuromag", the other way around.
This should be checked, right? Now we'll need to follow the names on the appendix:
https://docs.google.com/document/d/1HFUkAEE-pB-angVcYe6pf_-fVf4sCpOHKesUvfb8Grc/edit#heading=h.snhwsmgj62e1

**Comment by chrisfilo** [comment]
I think we should add those checks - especially considering that even experts (people involved in creation of BIDS MEG) made such mistakes. Whether this needs to happen before or after paper resubmission I don't have an opinion.

Would you like to open an issue at https://github.com/INCF/bids-validator/issues?

File: ds000117/.bidsignore [hunk 1]
```diff
@@ -1 +1,238 @@
-*_FLASH.*
\ No newline at end of file
+run-1_echo-1_FLASH.json
```
Comment (jasmainak) [inline ds000117/.bidsignore]: why don't you use wildcards

**Comment by jasmainak** [comment]
do you want me to update the validator for those checks before merging the PR? I would merge this PR if it passes the current version of the validator. Sorry I can't clone and check right now as my internet connection is too slow

@chrisfilo any reason there isn't a travis enabled on this repo which runs the validator on all the datasets? so we will know automatically if the changes are good or not

**Comment by jasmainak** [comment]
@guiomar I see from github: 529 lines added and 3,039,198 deleted. Is that what you wanted to do? Why is that?

File: ds000117/.bidsignore (hunk 1 above)
Comment (chrisfilo) [reply ds000117/.bidsignore]: +1

**Comment by chrisfilo** [comment]
@jasmainak there is no reason that integration is not implemented (it was actually suggested a long time ago see #5). It would be a valuable contribution.

**Comment by jasmainak** [comment]
Oh I could probably help with the travis yml file but I wouldn't know how to update the datasets to make the compliant if they don't pass. My MRI knowledge is a bit limited :-)

**Comment by chrisfilo** [comment]
All of the datasets in current BIDS-examples master are compatible and pass using latest validator release (I made sure about this a couple days ago).

**Comment by jasmainak** [comment]
I'm trying to resist being nerdsniped :-)

**Comment by chrisfilo** [comment]
Prioritizing works is an important skill to have... yet work would be pointless if you didn't have fun with it :)

**Comment by jasmainak** [comment]
okay, okay ;-) see https://github.com/INCF/BIDS-examples/pull/63

File: ds000117/.bidsignore (hunk 1 above)
Comment (guiomar) [reply ds000117/.bidsignore]: Ok! This was done by Rik, do you know his user in GitHub?

Comment (jasmainak) [reply ds000117/.bidsignore]: umm ... nopes. This one: https://github.com/RikHenson ?

**Comment by guiomar** [comment]
> @guiomar I see from github: 529 lines added and 3,039,198 deleted. Is that what you wanted to do? Why is that?

I just deleted everything that was on the ds000117 folder and replaced by Rik's new folder. I'll double check with him. If it was everything or just meg sessions.

> do you want me to update the validator for those checks before merging the PR? I would merge this PR if it passes the current version of the validator. Sorry I can't clone and check right now as my internet connection is too slow

I think it would be good, as Chris says even the experts can make this kind of mistakes.

Thanks a lot!

**Comment by jasmainak** [comment]
@guiomar did you run the validator on this? I get the following message:

> The directory ds000117/ failed an initial Quick Test. This means the basic names and structure of the files and directories do not comply with BIDS specification. For more info go to http://bids.neuroimaging.io/

**Comment by guiomar** [comment]
@jasmainak  Yes, I did, on my computer I only got this:
image

**Comment by guiomar** [comment]
And the error is due to empty niis.

File: ds000117/.bidsignore (hunk 1 above)
Comment (guiomar) [reply ds000117/.bidsignore]: Yes he is! :) 

@RikHenson

**Comment by RikHenson** [comment]
Sorry, I don't understand what the problem is or what you want me to do. I explained before that the nii files are empty, to save diskspace, so you will get those warnings. I don't know why file numbers have changed on github, but the tarball I sent you is correct and current, in sense that the files that generated it on my local version passed the online validator. Shall I re-send that tarball?

Do you want me to change Neuromag/Elekta and Elekta/Neuromag, or can you?

I don't understand the missing EOL in the .bidsignore - it is present in my local copy (that I tarballed)

R

**Comment by jasmainak** [comment]
@guiomar I am a bit lost, because I tried the same on your branch and it tells me "quicktest" fails. Actually I realized it's a problem with the validator but it's still weird that you get a different error. Are you sure you are trying the right url?

**Comment by jasmainak** [comment]
@RikHenson these are the files which have been removed: https://gist.github.com/jasmainak/f1907a2e6b225c2b007dc0ec2d116064 (the ones marked with D are deleted and the ones marked with M are modified). Do you confirm that the ones marked with D are not needed any more? It's not only .nii files but there are many other log files etc. which seemed to have been removed.

**Comment by guiomar** [comment]
@jasmainak It seemed that it wasn't updated properly. Try now?

**Comment by jasmainak** [comment]
Now I get 4 warnings but no errors, is it the same for you?

**Comment by guiomar** [comment]
I get the same as before

**Comment by jasmainak** [comment]
okay I figured out what's going on

@chrisfilo is it expected that we see more warnings when using the flag `--ignoreNiftiHeaders` with the validator? I would think there would be fewer warnings, no?

**Comment by RikHenson** [comment]
I've just realised why the number of files has decreased: in the original version, each fMRI volume was a separate 3D *.nii file, whereas now I merged them into a single 4D *.nii file (for each run).

I think the other changes you list Mainak are just because the files were moved around to respect the BIDS format.

**Comment by RikHenson** [comment]
PS When I tested my local copy with Mainak's online validator, I did get some warnings, yes, but no errors (when the *.nii files were real, rather than replaced with empty filenames in order to upload to github)

**Comment by jasmainak** [comment]
okay just wanted to make sure the files were not deleted accidentally :-) we'll probably have to do a squash in `bep_008` after the merge, but I guess @chrisfilo can take care of it in his PR.

File: ds000117/.bidsignore (hunk 1 above)
Comment (jasmainak) [reply ds000117/.bidsignore]: this has not yet been addressed

**Comment by guiomar** [comment]
Hi @jasmainak @RikHenson!
So everything it's ready here, right?
@chrisfilo could you merge this?

**Comment by chrisfilo** [comment]
I'll let @jasmainak to do the honors.

**Comment by RikHenson** [comment]
I don't really have time to check so many changes. Forgetting about new files and deleted files, has the content of any of the BIDS meta-data files (*.json, *.tsv, *.txt) changed?

**Comment by jasmainak** [comment]
merged after cosmit https://github.com/INCF/BIDS-examples/pull/45/commits/3a1c35f601f72118bf08c835b47e756cd48316e8

Thanks @guiomar and @RikHenson

**Pull request closed without merging**